test/commands/info/releasenotes/display.test.ts

```diff
@@ -264,3 +264,242 @@ describe('info:releasenotes:display', () => {
+    await runDisplayCmd(['-v', '4.5.6', '--hook']);
+
+    expect(getReleaseNotesStub.args[0][2]).to.equal('4.5.6');
+  });
+
+  it('calls getReleaseNotes with installed version if no arg is passed', async () => {
+    await runDisplayCmd([]);
+
+    expect(getReleaseNotesStub.args[0][2]).to.equal('3.3.3');
+  });
+
+  it('throws an error if getReleaseNotes lookup fails', async () => {
+    getReleaseNotesStub.throws(new Error('release notes error'));
+
+    try {
+      await shouldThrow(runDisplayCmd([]));
+    } catch (err) {
+      expect((err as Error).message).to.contain('release notes error');
+    }
+  });
+
+  it('parseReleaseNotes is called with the correct args', async () => {
+    await runDisplayCmd([]);
+
+    expect(parseReleaseNotesSpy.args[0]).to.deep.equal([
+      '## Release notes for 3.3.3',
+      '3.3.3',
+      mockInfoConfig.releasenotes.releaseNotesPath,
+    ]);
+  });
+
+  it('parser is called with tokens', async () => {
+    await runDisplayCmd([]);
+
+    const tokens = parseReleaseNotesSpy.returnValues[0] as marked.Token;
+
+    expect(markedParserSpy.calledOnce).to.be.true;
+    expect(markedParserSpy.args[0][0]).to.deep.equal(tokens);
+  });
+
+  it('logs markdown on the command line', async () => {
+    await runDisplayCmd([]);
+
+    expect(uxLogStub.args[0][0]).to.contain('## Release notes for 3.3.3');
+  });
+
+  it('throws an error if parsing fails', async () => {
+    try {
+      await shouldThrow(runDisplayCmd(['-v', '4.5.6']));
+    } catch (err) {
+      expect((err as Error).message).to.contain(
+        `Didn't find version '4.5.6'. View release notes online at: ${mockInfoConfig.releasenotes.releaseNotesPath}`
+      );
+    }
+  });
+
+  it('does not throw an error if --hook is set', async () => {
+    getReleaseNotesStub.throws(new Error('release notes error'));
+    const lifecycleStub = stubMethod(sandbox, Lifecycle.prototype, 'emitTelemetry');
+
+    await runDisplayCmd(['--hook']);
+    expect(lifecycleStub.calledOnce).to.be.true;
+    expect(uxWarnStub.args[0][0]).to.contain('release notes error');
+  });
+
+  it('renders a footer if --hook is set', async () => {
+    await runDisplayCmd(['--hook']);
+
+    expect(uxLogStub.args[1][0]).to.contain('to manually view the current release notes');
+  });
+
+  it('hides footer if env var is set', async () => {
+    getBooleanStub.withArgs('SF_HIDE_RELEASE_NOTES_FOOTER').returns(true);
+
+    const lifecycleStub = stubMethod(sandbox, Lifecycle.prototype, 'emitTelemetry');
+
+    await runDisplayCmd(['--hook']);
+    expect(lifecycleStub.calledOnce).to.be.true;
+    expect(uxLogStub.args[2]).to.be.undefined;
+  });
+
+  it('supports json output', async () => {
+    const json = await runDisplayCmd(['--json']);
+
+    const expected = {
+      body: `# Release notes for 'sf':${os.EOL}## Release notes for 3.3.3`,
+      url: mockInfoConfig.releasenotes.releaseNotesPath,
+    };
+
+    expect(json).to.deep.equal(expected);
+  });
+});
```

Comment:
I think a good chuck of these UTs don't need to be duplicated. Especially if we decide to check both env vars